File: Data Structures & Algorithms/duplicate-integer/submission-8.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        // store key value 
        HashMap<Integer, Integer> count = new HashMap<>();

        for(int number : nums){
            count.put(number, count.getOrDefault(number, 0) + 1);
        }

        for(int value : count.values()){
            if(value > 1){
                return true;
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-8.js
class Solution {
    topKFrequent(nums, k) {
        let map = new Map(); 
        for(let ch of nums){
            map.set(ch , (map.get(ch) || 0) + 1)
        }

      return [...map.keys()].sort((a , b ) => map.get(b) -  map.get(a)).slice(0 , k )
    }
}